book/tut01_firstfigure.ipynb

```diff
@@ -0,0 +1,522 @@
+{
```

Comment:
_Commented on notebook `book/tut01_firstfigure.ipynb` Cell 23 Line 2_
```markdown
Now, re-plot the figure with new colormap and ...
## Adding [`figure.colorbar`](https://www.pygmt.org/v0.13.0/api/generated/pygmt.Figure.colorbar.html)
```
TEST: Maybe it's better to capitalize "figure" here, i.e., "Figure.colorbar".